code/index.html

```diff
@@ -1,22 +1,89 @@
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="./style.css" />
+    <title>GAME ZONE NEWS</title>
   </head>
+
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+
+    <header>  <!--HEADER SECTION-->
+
+      <nav>
+        <img class="logo" src="./img/gamelogo.png" alt="GAME ZONE ICON">
+        <div class="left">
+          <a class="leftlinks" href="#">HOME</a>
+          <a class="leftlinks" href="#">GUIDE</a>
+        </div>
+        <div class="right">
+          <a class="rightlinks" href="#">UPCOMMING</a>
+          <a class="rightlinks" href="#">UPDATES</a>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <main class="bigger-news-container"> <!-- HEADLINE NEWS SECTION-->
+      <section class="big-news-container">
+        <img src="./img/psxbox.png" alt="PS5 XBOX Image">
+
+        <div class="big-news-text">
+          <article>
+            <h2>Playstation 5 v/s Xbox Series X !</h2>
+            <p>The next generation of consoles is coming, and Sony and Microsoft are getting ready to battle it out for system supremacy, pitting the PlayStation 5 against the Xbox Series X this holiday season.</p>
+          </article>
+          <a class="readmore" ref="https://www.tomsguide.com/face-off/ps5-vs-xbox-series-x">Read More...</a>
+        </div>
+      </section>
+    </main>
+
+    <hr/>
+
+    <h3 class="latestnews.">GAMES YOU SHOULD PLAY ...</h3>
+
+    <section class="newscards">  <!-- NEWS CARDS-->
+     
+      <div class="cards">
+        <img src="./img/rdr2.jpg" alt="Red Dead Redemption II">
+        <h3>RED DEAD REDEMPTION II</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/Diablo.jpg" alt="Diablo">
+        <h3>Diablo III</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/Witcher.jpeg" alt="Witcher">
+        <h3>Witcher III</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/gta5.jpg" alt="Grand Theft Auto 5">
+        <h3>Grand Theft Auto V</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/nfs.jpg" alt="Need For Speed">
+        <h3>Need for Speed - Uncut</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/Tombraider.jpg" alt="Tombraider">
+        <h3>Rise of Tombraider</h3>
+      </div>
+
+      <div class="cards">
+        <img src="./img/Watchdogs.jpg" alt="Watch Dogs">
+        <h3>Watchdogs 2</h3>
+      </div>
 
-    <section class="">
-      <!-- The other news  -->
     </section>
+
+    <Footer>
+      <p class="Footer">🦊 Game Zone News by Karan Mann 🦊</p>
+    </Footer>
```

Comment:
HTML elements are not case sensitive but it's a convention to write them in lowercase